packages/angular_devkit/build_optimizer/src/transforms/scrub-file.ts

```diff
@@ -50,21 +52,20 @@ function scrubFileTransformer(checker: ts.TypeChecker, isAngularCoreFile: boolea
           return ts.forEachChild(node, checkNodeForDecorators);
         }
         const exprStmt = node as ts.ExpressionStatement;
-        if (isDecoratorAssignmentExpression(exprStmt)) {
+        // Do checks that don't need the typechecker first and bail early.
+        if (isCtorParamsAssignmentExpression(exprStmt)
```

Comment:
consider swapping the two checks to optimize for the build speed of Ivy code over the legacy ctorParam format.

Reply:
Done